dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -1350,16 +1383,21 @@ synchronized String evaluateInterval() {
             }
 
             // Force some random variation into the pool size algorithm
-            poolAdjustment = forceVariation(poolSize, poolAdjustment, deltaCompleted, lowTput);
+            poolAdjustment = forceVariation(poolSize, poolAdjustment, deltaCompleted, lowActivity);
 
             // Format an event level trace point with the most useful data
             if (tc.isEventEnabled()) {
-                Tr.event(tc, "Interval data", toIntervalData(throughput, forecast, shrinkScore, growScore, poolSize, poolAdjustment));
+                Tr.event(tc, "Interval data", toIntervalData(throughput, forecast, deltaCompleted, shrinkScore, growScore,
+                                                             poolSize, poolAdjustment));
             }
 
             // Change the pool size and save the result, will check it at start of next control cycle
             targetPoolSize = adjustPoolSize(poolSize, poolAdjustment);
 
+        } catch (Exception ex) {
+            if (tc.isWarningEnabled()) {
+                Tr.warning(tc, "Exception caught: " + ex);
```

Comment:
warnings are translated messages.  If you are going to leave this in here you should add a translated message for it.  This will automatically get an FFDC added due to instrumentation which likely will be sufficient.

Reply:
There is a catch on the caller to this method so it should have made an FFDC for any exception as well?  Did you not see one?

Reply:
Mmmm ... I did not see an FFDC, but may have overlooked it. 
Guess I will delete this catch.